website_crm_quick_answer/data/base_automation_data.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="email_template" model="mail.template">
+            <field name="name">Quick response to website contact form</field>
+            <field name="model_id" ref="crm.model_crm_lead"/>
+            <field name="use_default_to" eval="True"/>
+            <field name="email_from">${object.company_id.email}</field>
+            <field name="lang">${object.env.context.get("lang")}</field>
+            <field name="subject">Thanks for your request</field>
+            <field name="body_html" type="html">
+                <table border="0" cellpadding="0" cellspacing="0" style="padding-top: 16px; background-color: #F1F1F1; font-family:Verdana, Arial,sans-serif; color: #454748; width: 100%; border-collapse:separate;">
```

Comment:
Why complicating the format? Why don't you use the general notification template + extra message?

Reply:
I copied the default template format in v12 to improve usability but I can return to the email without format, no problem.

Reply:
See the technique used in other notifications: take the standard notification template, and adds the main body content. This would be the preferred way, for customizing branding only in one place.

Reply:
OK👍🏻

Reply:
@pedrobaeza could you clarify this?